packages/patterns/src/keys/copySet.js

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck FIXME relies heavily on dynamic typing.
```

Comment:
Are these ts-nocheck directives going to preclude useful checks that would otherwise be possible?

Reply:
Possibly. When the rest of this settles I can come back and change them to local suppressions.